integrations/schemas/repository-configuration-schema.json

```diff
@@ -5,6 +5,9 @@
   "description": "The OSS-Review-Toolkit (ORT) provides a possibility to configure exclusions, resolutions and more in a file called `.ort.yml`. A full list of all available options can be found at https://github.com/oss-review-toolkit/ort/blob/main/docs/config-file-ort-yml.md.",
   "type": "object",
   "properties": {
+    "analyzer": {
```

Comment:
Nit: Commit message wrapping seem suboptimal in the rendered

```
The possibility to overwrite analyzer configurations within the
repository configuration was added in
07cc26d. Reflect this in the JSON
schema.
```

You should probably literally use the 7-digit abbreviated hash instead of the URL to fix this.